lib/chef/provider/systemd_unit.rb

```diff
@@ -121,9 +121,9 @@ def define_resource_requirements
       end
 
       action :reenable do
-        converge_by("reenabling unit: #{new_resource.unit_name}") do
+        converge_by("re-enabling unit: #{new_resource.unit_name}") do
```

Comment:
Any comments? just thought it would be nice converge msg?

Reply:
I think we should probably keep this since it better matches the action format

Reply:
sure